src/measure_theory/constructions/borel_space.lean

```diff
@@ -1496,6 +1496,27 @@ begin
   exact h's.closure_eq.symm
 end
 
+/-- The set of points for which a measurable sequence of functions converges is measurable. -/
+@[measurability] lemma measurable_set_exists_tendsto_at_top {ι : Type*}
+  [second_countable_topology α] [complete_space α] [nonempty ι] [encodable ι] [semilattice_sup ι]
```

Comment:
I've generalized it to polish spaces now but it was very painful and lean keeps inferring the wrong topology, in particular lean keeps infers the original topology rather than the new one generated by `upgrade_polish_space α`. 
Is there a way around this?